tests/general/ShebangBearTest.py

```diff
@@ -0,0 +1,61 @@
+import os
+from queue import Queue
+
+from bears.general.ShebangBear import ShebangBear
+from coalib.settings.Section import Section
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.results.Result import RESULT_SEVERITY, Result
+
+
+def get_testfile_path(name):
+    return os.path.join(os.path.dirname(__file__),
+                        'shebang_test_files',
+                        name)
+
+
+def load_testfile(name):
+    with open(get_testfile_path(name)) as f:
+        file_lines = f.readlines()
+    return file_lines
+
+
+class ShebangBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.section = Section('')
+        self.uut = ShebangBear(self.section, Queue())
+
+    def test_basic_usage(self):
+        self.check_validity(self.uut,
+                            ['#!/usr/bin/env python \n print hello world'])
+        self.check_validity(self.uut,
+                            [''])
```

Comment:
I think an empty file should be written as an empty list, that's how python handles it :+1:

Reply:
Maybe you should put those into test files too? Up to you :)